libvirt/tests/src/incremental_backup/incremental_backup_lifecycle.py

```
@@ -209,6 +213,94 @@ def test_kill_qemu_during_libvirtd_restart(test, params, backup_file_list):
     return backup_file_list
 
 
+def test_shutdown_vm(test, params, backup_file_list):
```

Ruff (0.14.13)

Unused function argument: `backup_file_list`

(ARG001)